webiny/webiny-js
# Introduced the brand new "Files" field

## Related Issue
Closes #850 
Headless CMS - File field 

## Your solution
Basically followed the solution mentioned in the issue

## How Has This Been Tested?
Manually, by performing the following steps:

- Visit the admin app and head over to the `content model` section

- Create a brand new content model with the `file field ` (you can also toggle `multiValues` option in the field settings)

- Head over to the content entry section and create a new content entry with file

- User should be able to select and upload file

## Screenshots :
https://www.loom.com/share/f776c2d44ac54b978fcb1cc1bb993f9c
